### ISSUE TITLE
Error popup upon opening Skill tab, doesn't crash PoB

### ISSUE THREAD

**Issue by Oogablast** [PathOfBuildingCommunity/PathOfBuilding#1859]
Thank you for reporting an issue.

Before reporting a bug, **please make sure that you are running the latest version.**
You can get the latest version number from `CHANGELOG.md`

Please try to fill in as much of the template below as you're able.

* **What happened?**
* Switched over to Skills tab
* **What were you trying to do?**
* Open Skills tab
* **What steps will reproduce the bug?**
* At the time I don't recall the exact settings but simply opening up the skills tab would cause the error message to appear
* **Does it reproduce every time?**
* No
* **Provide a build code:**
* https://pastebin.com/u96B81Sr
* Modified since error first appeared
Always provide a build code that exhibits the bug you want to report, even if it is not specific to a particular build. This helps us greatly to reproduce bugs faster. Otherwise, we will have to ask you for the code first before investigating, which would delay any possible fixes.
* **Screenshots:**
* 
![image](https://user-images.githubusercontent.com/53131972/104467457-55e44780-5584-11eb-9093-1c33ba9b739e.png)

If you receive an error or experience graphical bugs, include one or multiple screenshots. GIFs or short videos are also okay, if the bug cannot be displayed in a still image.

**Comment by Wires77** [comment]
How did you make this build?  From scratch on the new release?  If you change the quality for Herald of Ash and then change it back to default, it will fix the issue.  Not sure how it got into a bad state to begin with, though.

**Comment by Wires77** [comment]
Fixed by #1897

**Comment by trollkind** [comment]
Don't know if this helps and if it was a general fix that should work for every similar case but I get that popup when I mouse-over the Blasphemy gem. My other gems are fine, only tested the ones on my build though.
1.4.170.22

**Comment by Wires77** [comment]
> 
> 
> Don't know if this helps and if it was a general fix that should work for every similar case but I get that popup when I mouse-over the Blasphemy gem. My other gems are fine, only tested the ones on my build though.
> 1.4.170.22

Do you have a build we can test with?  I can't reproduce this.

**Comment by trollkind** [comment]
https://pastebin.com/H9kJ7MTM

It's my old Glacial Hammer build that I dragged through the last three leagues. This skill group once had a Blasphemy in it but I deleted it after Assasin's Mark got reworked. Now for my Elementalist variant I wanted to add Blasphemy again for Frostbite.
It seems to be only this Blasphemy gem though. When I add a new skill group and a new Blasphemy, there's no problems, so only the one in the "Auras" group is bugged. Sorry, should have tested more, I know better. So it's probably not a big issue and similar to the original report here instead of gem specific.
